railties/lib/rails/application/finisher.rb

```diff
@@ -231,6 +231,12 @@ def self.complete(_state)
           ActiveSupport::DescendantsTracker.disable_clear!
         end
       end
+
+      initializer :enable_yjit do
```

Comment:
Might want to defer this to `after_initialize` to avoid compiling too much boot only code.

Reply:
That is after `after_initialize`.  In fact that is the last thing it happens so it is even later than it was before.

Reply:
Ah, good point, I missed it was in `finisher.rb` 👍

Reply:
`after_initialize` runs on line 94 of this file, since this initializer is after the initializer that runs all the `after_initialize` blocks this will run after all initialization code.